plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -156,7 +159,13 @@ func (c *MaxPDVolumeCountChecker) filterVolumes(volumes []api.Volume, namespace
 			}
 			pvc, err := c.pvcInfo.GetPersistentVolumeClaimInfo(namespace, pvcName)
 			if err != nil {
-				return err
+				// if the PVC is not found, log the error and count the PV towards the PV limit
+				// generate a random volume ID since its required for de-dup
+				glog.Error(err)
```

Comment:
it might not be a bad idea here to make this an Errorf and say something like
`glog.Errorf("Unable to look up PVC info for %s/%s, assuming PVC matches predicate when counting limits: %v", namespace, pvcName, err)`, that way we can see what the context of the error is from the logs.